dhall/src/Dhall/Map.hs

```diff
@@ -491,3 +492,12 @@ toMap (Map m _) = m
 keys :: Map k v -> [k]
 keys (Map _ ks) = ks
 {-# INLINABLE keys #-}
+
+{-| Return the values from a `Map` in their original order
+
+>>> values (fromList [("B",1),("A",2)])
+[1,2]
+-}
+values :: Ord k => Map k v -> [v]
```

Comment:
Perhaps call this one `elems` following the example of [Data.Map](https://hackage.haskell.org/package/containers-0.5.11.0/docs/Data-Map-Strict.html#v:elems)?